File: LC/217_contains_duplicate.js
class Solution {
  /**
   * @param {number[]} nums
   * @return {boolean}
   */
  hasDuplicate(nums) {
    nums.sort((x, y) => x - y);
    for (let i = 1; i < nums.length; ++i) {
      if (nums[i - 1] === nums[i]) return true;
    }
    return false;
  }
}

File: LC/217_contains_duplicate.cpp
#include <vector>
#include <algorithm>

class Solution
{
public:
  bool hasDuplicate(std::vector<int> &nums)
  {
    std::sort(nums.begin(), nums.end());
    for (size_t i = 1; i < nums.size(); ++i)
    {
      if (nums[i - 1] == nums[i])
      {
        return true;
      }
    }
    return false;
  }
};